kernel/aster-nix/src/syscall/sched_getaffinity.rs

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use core::mem;
+
+use super::SyscallReturn;
+use crate::{
+    prelude::*,
+    process::{process_table, Pid},
+    util::write_val_to_user,
+};
+
+pub fn sys_sched_getaffinity(
+    pid: Pid,
+    cpuset_size: usize,
+    user_mask_ptr: Vaddr,
```

Comment:
Rename this variable to `cpu_set_ptr` to be consistent with the type `cpu_set_t`. Rename the `result` variable to `cpu_set` or `dummy_cpu_set` for the same reason.

Reply:
Done.